SimCalorimetry/HcalTrigPrimProducers/python/hcaltpdigi_cff.py

```diff
@@ -4,12 +4,9 @@
 
 import FWCore.ParameterSet.Config as cms
 
-from Configuration.Eras.Modifier_run2_HCAL_2018_cff import run2_HCAL_2018
-from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA
-
-from SimCalorimetry.HcalTrigPrimProducers.hcaltpdigi_cfi import *
-from CalibCalorimetry.CaloTPG.CaloTPGTranscoder_cfi import *
```

Comment:
Evidently the configurations in these packages are even more convoluted than what was visible in this particular case.
I have verified that by putting back this line
```PYTHON
from CalibCalorimetry.CaloTPG.CaloTPGTranscoder_cfi import *
```
the now failing workflows succeed (at least, the ones I tested). Quite likely other configs link this `hcaltpdigi_cff.py` in order to import CaloTPGTranscoder_cfi.

I think that at some point some further cleanup should be applied in these packages, but not in this PR.
For the moment it can be enough to put this line back and restart the tests